showcase/src/app/demo/typography/demo-typography.component.html

```diff
@@ -29,10 +31,14 @@ <h4>Headline 4</h4>
 		<p class="mat-caption">Mat-Caption</p>
 	</example-viewer>
 </section>
+
 <section class="stark-section">
 	<h1 translate>SHOWCASE.DEMO.TYPOGRAPHY.CUSTOMIZATION.TITLE</h1>
 	<p translate [innerHTML]="'SHOWCASE.DEMO.TYPOGRAPHY.CUSTOMIZATION.DETAILS' | translate"></p>
 	<stark-pretty-print [data]="customizeMap" format="scss" [enableHighlighting]="true"></stark-pretty-print>
+	<p class="mt3" translate>SHOWCASE.DEMO.TYPOGRAPHY.CUSTOMIZATION.CUSTOM_FONT_EXPLAINED</p>
+	<stark-pretty-print [data]="fontFaceExample" format="scss" [enableHighlighting]="true"></stark-pretty-print>
+	<p class="mt3" [innerHTML]="'SHOWCASE.DEMO.TYPOGRAPHY.CUSTOMIZATION.CDN_FONT_EXPLAINED' | translate"></p>
```

Comment:
These new paragraphs are not in the right place I think, I suggest to put them right before "Available classes & tags", as part of the "Font Family" section. What do you think?

Then we should adapt a bit the text in the "Customization" section because it will be duplicated then, I mean this part:

> Stark allows you to customize simultaneously the typography settings of Angular Material and Stark easily by adding a simple SCSS map declaration. 
First, declare the SCSS map and then you need to import the Stark's theme. On this example, a full version of the map is shown but you can also use it partially.

Reply:
It does fit better there, I moved it and reworked the text a bit. Instead of changing the customization text I referred to it. Is this also acceptable?